test/write-batch.js

```diff
@@ -294,6 +294,21 @@ describe('batch support', function() {
       });
   });
 
+  it('cannot append to committed batch', function() {
```

Comment:
Can you add a test that verifies that you can call commit() multiple times if that is the contract we're offering?

Reply:
Done.